src/components/dialogs/equipment-deletion-dialog.js

```diff
@@ -235,11 +165,11 @@ const EquipmentDeletionDialog = ({
                             variant="filled"
                             fullWidth
                         >
-                            {Object.values(equipmentTypes).map((values) => {
+                            {Object.values(EQUIPMENT_TYPES).map((values) => {
```

Comment:
There is more values in EQUIPMENT_TYPES object than the previous equipmentTypes object, for example SWITCH, and removing a switch is not implemented ...

Reply:
done